docs/lua/script.lua

```
@@ -0,0 +1,20 @@
+function parse_log(slog)
+    local json = require("json")
+
+    -- Use json.decode to parse the string and have a valid lua table
+    data = json.decode(slog)
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'data'

(W111)